### PR TITLE
Chore update dg addmeeting

### PR DESCRIPTION
**Pull request by junweimoo**

### CODE REVIEW

**Review by sikai00 — DISMISSED** [review]
looks good, however we will have to standardize on the dg format before actual submission!

File: docs/diagrams/AddMeetingSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+storage --[STORAGE_COLOR]> logic
+deactivate storage
+
+logic --[LOGIC_COLOR]> ui
+deactivate logic
+
+ui--[UI_COLOR]> user
+deactivate ui
+
+@enduml
```
Comment (ThomasHoooo) [inline docs/diagrams/AddMeetingSequenceDiagram.puml:45]: Please add a new line at eof

**Review by ThomasHoooo — COMMENTED** [review]
Please make sure CI/CD passes before merging

**Review by ThomasHoooo — APPROVED** [review]
LGTM

**Pull request merged**